src/applications/gi/components/profile/HeadingSummary.jsx

```diff
@@ -81,9 +81,18 @@ class HeadingSummary extends React.Component {
           </div>
           <div>
             <div className="usa-width-one-half medium-6 small-12 column">
-              <IconWithInfo icon="map-marker" present={it.city && it.country}>
-                {it.city}, {it.state || it.country}
-              </IconWithInfo>
+              {it.country === 'USA' ? (
```

Comment:
This variable name is not great

Reply:
Agreed. If I made more changes to the file, I would have fixed it. Didn't want to cause more potential issues. :)